pyenvbuilder/commands/interface.py

```diff
@@ -0,0 +1,20 @@
+'''
+Blueprint for the commands classes
+'''
+
+
+class Command(object):
+    def __init__(self, name, summary):
+
+        self.name = name
+        self.summary = summary
+
+    @staticmethod
```

Comment:
Let's remove the `@staticmethod ` from here and `run`. Despite the fact that they can be static not always they will be.